pkg/archive/archive.go

```diff
@@ -159,6 +173,7 @@ func CompressStream(dest io.WriteCloser, compression Compression) (io.WriteClose
 	}
 }
 
+// Extension returns the extend file name of the compression algorithm.
```

Comment:
"Extension returns the extension of a file that uses the specified compression algorithm" maybe :blush:.